### PR TITLE
Poc to reuse MenuBlock

### PR DESCRIPTION
**Pull request by VincentLanglet**
## Subject

I am targeting this branch, because {reason}.

Closes #{put_issue_number_here}.

### CODE REVIEW

**Comment by jordisala1991** [comment]
This should also be reverted: https://github.com/sonata-project/SonataPageBundle/blob/01f5a86138733dfbc28aafcf05d8dd52d88f4e8f/src/Resources/config/block.php#L47

**Comment by VincentLanglet** [comment]
> This should also be reverted:
> 
> https://github.com/sonata-project/SonataPageBundle/blob/01f5a86138733dfbc28aafcf05d8dd52d88f4e8f/src/Resources/config/block.php#L47

Test are not failing ^^' 

Are you talking about this revert https://github.com/sonata-project/SonataPageBundle/pull/1612/files#diff-b50ae12595c81d84839724f494ff3fc84c2c4bfac81990d3e14c87ecc3459ce2L47 ?

**Comment by jordisala1991** [comment]
> > This should also be reverted:
> 
> > 
> 
> > https://github.com/sonata-project/SonataPageBundle/blob/01f5a86138733dfbc28aafcf05d8dd52d88f4e8f/src/Resources/config/block.php#L47
> 
> 
> 
> Test are not failing ^^' 
> 
> 
> 
> Are you talking about this revert https://github.com/sonata-project/SonataPageBundle/pull/1612/files#diff-b50ae12595c81d84839724f494ff3fc84c2c4bfac81990d3e14c87ecc3459ce2L47 ?

Yes, if you are not passing the block on the parent, how is this working?

**Comment by VincentLanglet** [comment]
> Yes, if you are not passing the block on the parent, how is this working?

I don't understand, I remove the constructor param https://github.com/sonata-project/SonataPageBundle/pull/1612/files#diff-f0461df8f4a53195f2b565ad37746df1e8076ff748edb18c3e2689581f2db4caL40 so I also remove in the config.

Now the block is extending BaseBreadcrumbService in which the constructor signature is reverted by 
https://github.com/sonata-project/SonataSeoBundle/pull/692

File: src/Resources/config/block.php [hunk 1]
```diff
@@ -44,7 +44,6 @@
             ->tag('sonata.breadcrumb')
             ->args([
                 new ReferenceConfigurator('twig'),
-                new ReferenceConfigurator('sonata.block.service.menu'),
```
Comment (jordisala1991) [inline src/Resources/config/block.php:47]: didnt saw this line, all fine.

**Comment by jordisala1991** [comment]
@VincentLanglet I tried this PR (that uses at the same time the 3 PRs and everything seems fine).

File: src/Block/ChildrenPagesBlockService.php [hunk 2]
```diff
@@ -123,23 +123,28 @@ public function configureEditForm(FormMapper $form, BlockInterface $block): void
                 ['title', TextType::class, [
                   'required' => false,
                     'label' => 'form.label_title',
```
Comment (jordisala1991) [inline src/Block/ChildrenPagesBlockService.php:125]: is this line one space off?

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**